google-account-plugin/src/com/google/cloud/tools/intellij/stats/UsageTracker.java

```diff
@@ -16,99 +16,64 @@
 
 package com.google.cloud.tools.intellij.stats;
 
-
-import org.jetbrains.annotations.NotNull;
-import org.jetbrains.annotations.Nullable;
-
 /**
  * Interface for defining the actual tracking behavior, implementations must be declared in
  * plugin.xml for the {@link UsageTrackerExtensionPointBean} extension point.
  */
 public interface UsageTracker {
 
   /**
-   * When tracking events, do NOT include any information that can identify the user.
-   */
-  void trackEvent(@NotNull String eventCategory,
-      @NotNull String eventAction,
-      @Nullable String eventLabel,
-      @Nullable Integer eventValue);
-
-  /**
    * Returns a fluent API for pinging tracking events.
    *
-   * <p>Example:
-   * {@code
-   *  trackEvent("com.google.cloud.tools")
-   *    .withAction("appengine.deployment")
-   *    .andLabel("flex")
-   *    .setValue(1)
-   *    .send();
-   * }
+   * <p>Example: {@code sendEvent("appengine.deployment").withLabel("flex").setValue(1).ping();}
    *
-   * @param category this is used to identify the client being used. It will be typically the same
-   *                 for all invocations from the same plugin.
-   * @return a fluent interface that allows the user to set the 'action' being tracked by the event
+   * @param action is typically a specific operation the user has performed in the plugin, and is
+   *               often prefixed with a domain such as 'appengine.' or 'clouddebugger.'
+   * @return a fluent interface for setting the remaining parameters of a tracking ping
    */
-  //TODO: Check all invocations fo track event and consider defaulting 'category' to a global value.
-  PartialTrackingEventAction trackEvent(String category);
+  PartialTrackingEventLabel trackEvent(String action);
```

Comment:
Do we have naming issue with `PartialTrackingEventLabel`? My concern is that given this: 

```
    UsageTrackerProvider.getInstance().trackEvent(GctTracking.APP_ENGINE_STOP).ping();
```
Notice that track event returns a "partial" label yet its totally legitimate to send a ping from this. Note this is only a naming concern. Maybe its OK because it is also a `PingsAnalytics`..